pkg/Library/CoreXenon/Framework0/Id.js

```diff
@@ -0,0 +1,18 @@
+
+/**
+ * @license
+ * Copyright 2023 NeonFlan LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
```

Comment:
this is `Framework0` - is this intentional?

Reply:
Nope, ty.